### PR TITLE
Finished NeZha-D1 and D1_LicheeRV test task

### PR DESCRIPTION
**Pull request by Sebastianhayashi**
Update test result of  NeZha-D1 and D1_LicheeRV

### CODE REVIEW

File: .obsidian/app.json
Comment (KevinMX) [inline .obsidian/app.json:1]: Remove these junk files.
Consider update .gitignore

File: .obsidian/appearance.json
Comment (KevinMX) [inline .obsidian/appearance.json:1]: Same here.

File: .obsidian/app.json
Comment (KevinMX) [inline .obsidian/app.json:1]: Remove these junk files.
Consider update .gitignore

File: .obsidian/core-plugins-migration.json
Comment (KevinMX) [inline .obsidian/core-plugins-migration.json:1]: Same.

File: NeZha-D1s/README.md [hunk 1]
```diff
@@ -4,22 +4,27 @@
...
-- Tina Linux
-  - 下载链接：链接：https://pan.baidu.com/s/1v55AKMFripaEu22tJ92lmw?pwd=awol 提取码：awol
+- Tina Linux:
+  - 下载链接: https://pan.baidu.com/s/1v55AKMFripaEu22tJ92lmw?pwd=awol 提取码：awol
   - 参考安装文档：https://d1s.docs.aw-ol.com/study/study_1tina/
 - RT-Thread:
   - 下载链接：https://github.com/RT-Thread/rt-thread
   - 参考安装文档：https://github.com/RT-Thread/rt-thread/blob/master/bsp/allwinner/d1s/README-M7.md
+  - Ubuntu:
+  - 下载链接：https://cdimage.ubuntu.com/releases/noble/release/ubuntu-24.04-preinstalled-server-riscv64+nezha.img.xz?_gl=1*1x36sbc*_gcl_au*NDI5ODM0MDYxLjE3MTgyNjU3NzE.&_ga=2.251705994.2059181571.1718425068-1272776944.1718425068
```
Comment (KevinMX) [inline NeZha-D1s/README.md]: Why so much garbage after .xz?
I assume these are for tracking / data collection / metrics. Better remove them.

File: NeZha-D1s/README.md [hunk 2]
```diff
@@ -4,22 +4,27 @@
...
+  - 参考安装文档：https://wiki.ubuntu.com/RISC-V/Nezha%20D1?_gl=1*1x36sbc*_gcl_au*NDI5ODM0MDYxLjE3MTgyNjU3NzE.
```
Comment (KevinMX) [inline NeZha-D1s/README.md]: Same here. Never seen these on Ubuntu sites before. Maybe it's your browser.

File: NeZha-D1s/Ubuntu/README.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
+# Ubuntu 23.10 VisionFive 2 版本测试报告
```
Comment (KevinMX) [inline NeZha-D1s/Ubuntu/README.md:1]: 23.10 -> 24.04?

File: NeZha-D1s/Ubuntu/README.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+
+## 测试环境
+
+### 操作系统信息
+
+- 系统版本：Ubuntu 24.04
+- 下载链接：https://cdimage.ubuntu.com/releases/noble/release/ubuntu-24.04-preinstalled-server-riscv64+nezha.img.xz?_gl=1*1x36sbc*_gcl_au*NDI5ODM0MDYxLjE3MTgyNjU3NzE.&_ga=2.251705994.2059181571.1718425068-1272776944.1718425068
```
Comment (KevinMX) [inline NeZha-D1s/Ubuntu/README.md]: Same here. Check your URLs.

File: NeZha-D1s/Ubuntu/README.md [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+- 参考安装文档：https://wiki.ubuntu.com/RISC-V/Nezha%20D1?_gl=1*1x36sbc*_gcl_au*NDI5ODM0MDYxLjE3MTgyNjU3NzE.
```
Comment (KevinMX) [inline NeZha-D1s/Ubuntu/README.md]: Same.

**Review by KevinMX — CHANGES_REQUESTED** [review]

File: NeZha-D1s/Ubuntu/README.md (hunk 3 above)
Comment (Sebastianhayashi) [reply NeZha-D1s/Ubuntu/README.md:1]: VF2 doesn't have the test result, I just created a README prepare for the new result,I'll change it and point out that still no result

File: NeZha-D1s/Ubuntu/README.md [hunk 6]
```diff
@@ -5,8 +5,8 @@
 ### 操作系统信息
 
 - 系统版本：Ubuntu 24.04
-- 下载链接：https://cdimage.ubuntu.com/releases/noble/release/ubuntu-24.04-preinstalled-server-riscv64+nezha.img.xz?_gl=1*1x36sbc*_gcl_au*NDI5ODM0MDYxLjE3MTgyNjU3NzE.&_ga=2.251705994.2059181571.1718425068-1272776944.1718425068
-- 参考安装文档：https://wiki.ubuntu.com/RISC-V/Nezha%20D1?_gl=1*1x36sbc*_gcl_au*NDI5ODM0MDYxLjE3MTgyNjU3NzE.
+- 下载链接：https://cdimage.ubuntu.com/releases/noble/release/ubuntu-24.04-preinstalled-server-riscv64+nezha.img.xz
+- 参考安装文档：https://wiki.ubuntu.com/RISC-V/Nezha%20D1?
```
Comment (KevinMX) [inline NeZha-D1s/Ubuntu/README.md:9]: ?

**Review by KevinMX — CHANGES_REQUESTED** [review]

**Pull request merged**